### ISSUE TITLE
License

### ISSUE THREAD

**Issue by jselikof** [turner/HelloMetal#41]
Thanks for these examples. Just checking if a license might be added in the future?

**Comment by turner** [comment]
Good idea. I haven't done much with this project in a while. I can add creative commons or something similar.
